config/routes.rb

```diff
@@ -131,6 +131,12 @@
     end
   end
 
+  namespace :admin do
```

Comment:
@mjgiarlo What do you mean in terms of namespacing?  admin is the same namespace isn't it?